content/2017/1/25/parallax.rst

```diff
@@ -0,0 +1,36 @@
+public: yes
+author: sondra
+tags: [design]
+image:
+  - src: '2017/parallax/parallax.jpg'
+summary: |
+  Whether you like it not, parallax scrolling is a reality, so let's learn how to do it well with CSS 3D. Paul Lewis' guide to performant parallaxing is the perfect place to start.
+
+
+Let's Talk About Parallax: Dos and Don'ts
+=========================================
+
+First, I'd like to thank `Rachel Nabors`_, award winning cartoonist and digital storyteller, for pointing us to Paul Lewis' excellent `article on performant parallaxing`_.
```

Comment:
I think this can be a simpler (and less prominent?) "Thanks to Rachel Nabors for pointing us..."